spras/containers.py

```diff
@@ -41,6 +42,79 @@ def convert_docker_path(src_path: PurePath, dest_path: PurePath, file_path: Unio
     rel_path = file_path.relative_to(src_path)
     return PurePosixPath(dest_path, rel_path)
 
+def download_gcs(gcs_path: str, local_path: str, is_dir: bool):
+    # check that output path exists
+    if not os.path.exists(Path(local_path).parent):
+        os.makedirs(Path(local_path).parent)
+
+    # build command
+    cmd = 'gcloud storage'
+    # rsync with checksums to make file transfer faster for larger files
+    cmd = cmd + ' rsync --checksums-only'
+    # check if directory
+    if is_dir:
+        cmd = cmd + ' -r'
+    cmd = cmd + ' ' + gcs_path + ' ' + local_path
+
+    print(cmd)
+    # run command
+    subprocess.run(cmd, shell=True)
+
+    if is_dir and Path(Path(local_path)/'gcs_temp.txt').exists():
+        os.remove(Path(Path(local_path)/'gcs_temp.txt'))
```

Comment:
```suggestion
        Path(Path(local_path)/'gcs_temp.txt').unlink()
```
Does this work? It's a bit simpler to me to stick with pathlib operations.

Reply:
This works.